interpreter/valid/valid.ml

```diff
@@ -98,13 +98,13 @@ let refer_func (c : context) x = refer "function" c.refs.Free.funcs x
 
 let cont_type_of_heap_type (c : context) (ht : heap_type) at : cont_type =
   match ht with
-  | DefHT dt -> as_cont_str_type (expand_def_type dt)
+  | DefHT dt -> assert false
```

Comment:
Nit: just remove this case, it's subsumed by the default below.

Reply:
Thanks! I accidentally forgot to remove this redundant case.